Visi.Repository/PatientQueryFactory.cs

```diff
@@ -29,9 +29,12 @@ protected override IQueryable<ViSiPatient> HandleShapes(IQueryable<ViSiPatient>
                 return source;
 
             var sorted = source;
-            foreach (var sortOp in SortOperations)
+            if (SortOperations.HasAny())
```

Comment:
Performance improvement?

Reply:
Null check. Foreach crashes on null collection.
HasAny() checks for null and not empty.